features/draganddrop.yml

```
@@ -0,0 +1,41 @@
+name: Drag and Drop
+description: "The Drag and Drop API lets web applications add dragging and dropping of elements and external files such as images onto web pages. Developers can customize which elements can become draggable, the type of feedback the draggable elements produce, and the droppable elements."
```

```suggestion
description: "The Drag and Drop API lets users drag and drop elements and external files such as images onto web pages. Developers can customize which elements can become draggable, the type of feedback the draggable elements produce, and the droppable elements."
```

I don't think we can or need to try to escape the circular definition here. I noted MDN uses `drag-and-drop`, but that doesn't seem to be the preferred term per Google Trends. https://trends.google.com/trends/explore?date=today%205-y&q=drag-and-drop,drag%20and%20drop&hl=en